libnetwork/internal/rootlessnetns/netns_linux.go

```diff
@@ -31,8 +32,8 @@ const (
 	// refCountFile file name for the ref count file
 	refCountFile = "ref-count"
 
-	// rootlessNetNsSilrp4netnsPidFile is the name of the rootless netns slirp4netns pid file
-	rootlessNetNsSilrp4netnsPidFile = "rootless-netns-slirp4netns.pid"
+	// rootlessNetNsConnPidFile is the name of the rootless netns slirp4netns/pasta pid file
+	rootlessNetNsConnPidFile = "rootless-netns-slirp4netns.pid"
```

Comment:
might as well rename the file, this code was only add during 5.0-dev so it is fine to break this here